test/asyncio_tests/__init__.py

```diff
@@ -123,10 +123,9 @@ def asyncio_rsc(self, uri=None, *args, **kwargs):
             *args,
             **self.get_client_kwargs(**kwargs))
 
-    @asyncio.coroutine
-    def make_test_data(self):
-        yield from self.collection.delete_many({})
-        yield from self.collection.insert_many([{'_id': i} for i in range(200)])
```

Comment:
Do we need to test that `yield` still works? Perhaps we should add some tests for `yield` specifically?

Reply:
Yeah forget this comment.